<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <exportmacro>PLUGIN_SURFACE_DEFORMATION_EXPORT</exportmacro>
 <class>SurfaceDeformation_TabWidget</class>
 <widget class="QWidget" name="SurfaceDeformation_TabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QComboBox" name="combo_positionAttribute">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>- select attribute -</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="15" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>156</width>
       <height>161</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QComboBox" name="combo_handleVertexSet">
     <item>
      <property name="text">
       <string>- select set -</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Position :</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Free vertex set :</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QComboBox" name="combo_freeVertexSet">
     <item>
      <property name="text">
       <string>- select set -</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="QPushButton" name="button_startStop">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Handle vertex set :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QListWidget" name="list_maps"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>combo_positionAttribute</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
